html/captive-portal/templates/login.html

```diff
@@ -70,8 +70,15 @@
           <div class="input">
             <span>[% i18n("Password") %]</span><input class="field" name="password" type="password" />
           </div>
+
           [% END %]
-          
+          [% FOREACH field IN mandatory_fields %]
+          <div class="input">
+            <span>[% i18n("$field") %]</span>
+            <input class="field" name="[%field%]" type="text" value="" /><br/>
```

Comment:
Yes you are true, i just have a case with a local auth source and i need to have mandatory field email.
So i have probably have to add a checkbox in the auth source configuration to ask if we have to take care of mandatory fields on the portal profile or no.